### PR TITLE
Apply translations from web categories.txt

### PR DESCRIPTION
**Pull request by teolemon**
Apply translations from web

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -1708,21 +1780,22 @@ en:distilled anis
...
 fr:Tequilas
 hu:Tekilák
 it:Tequila
 nl:Tequilas, Tequila
 nl_be:Tequilas
 ro:Tequilla
+el, el:Ouzo, Oύζο
+protected_name_file_number:en: PGI-CY+GR-01828
+protected_name_type:en: gi
+origins:en: en:Cyprus, Greece
```
Comment (Ban3) [inline taxonomies/categories.txt:1798]: Doesn't look like it belongs under Tequila.

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -30846,10 +31568,10 @@ country:en:Italy
 
 <en:Italian cheeses
 it:Fontina
-country:en:Italy
-#wikidata:en:
 en:Fontina cheese from cow's milk
 fr:Fontina
+country:en:Italy
+#wikidata:en:
```
Comment (Ban3) [inline taxonomies/categories.txt:31574]:
```suggestion
```

**Comment by Ban3** [comment]
```txt
2020-08-30T22:25:54.4858197Z Errors in the categories taxonomy definition:
2020-08-30T22:25:54.4956367Z ERROR - de:weißer-nougat already is a synonym of de:weißer-nougat for entry en:white-nougat - de:spanischer-weißer-nougat cannot be mapped to entry en:toasted-egg-yolk-turron
2020-08-30T22:25:54.5043890Z ERROR - es:caviar already is a synonym of es:huevas for entry en:fish-eggs - es:caviar cannot be mapped to entry en:caviars
2020-08-30T22:25:54.5045264Z ERROR - it:caviale already is a synonym of it:uova-di-pesce for entry en:fish-eggs - it:caviale cannot be mapped to entry en:caviars
2020-08-30T22:25:54.5046107Z ERROR - he:מוצרי-חלב already is a synonym of he:חלב for entry en:dairies - he:מוצרי-מזון-מותססים cannot be mapped to entry en:fermented-milk-products
2020-08-30T22:25:54.5061343Z ERROR - es:leche-condensada already is a synonym of es:leche-condensada for entry en:condensed-milks - es:leches-evaporadas cannot be mapped to entry en:evaporated-milks
2020-08-30T22:25:54.5062117Z ERROR - it:frutta-a-guscio already is a synonym of it:noci for entry en:nuts - it:frutta-a-guscio cannot be mapped to entry en:shelled-nuts
2020-08-30T22:25:54.5062953Z ERROR - fi:vapaan-kanan-munat already is a synonym of fi:vapaan-kanan-munat for entry en:free-range-chicken-eggs - fi:lattiakanalan-munat cannot be mapped to entry en:barn-chicken-eggs
2020-08-30T22:25:54.5063619Z ERROR - he:מדגנים already is a synonym of he:מדגנים for entry en:cereal-grains - he:דגני-בוקר cannot be mapped to entry en:breakfast-cereals
2020-08-30T22:25:54.5064536Z ERROR - it:cereali already is a synonym of it:cereali-e-i-loro-prodotti for entry en:cereals-and-their-products - it:cornflakes cannot be mapped to entry en:corn-flakes
2020-08-30T22:25:54.5065327Z ERROR - de:gepufftes-getreide already is a synonym of de:puffkörner for entry en:puffed-grains - de:gepufftes-getreide cannot be mapped to entry en:puffed-cereals
2020-08-30T22:25:54.5072725Z ERROR - ru:специи already is a synonym of ru:вкусовые-добавки for entry en:condiments - ru:специи cannot be mapped to entry en:spices
2020-08-30T22:25:54.5083862Z ERROR - it:brodo already is a synonym of it:brodo for entry en:broths - it:brodo-liquido cannot be mapped to entry en:liquid-broths
2020-08-30T22:25:54.5108252Z ERROR - de:spiralnudeln already is a synonym of de:spiralnudeln for entry en:torti - de:fusilli cannot be mapped to entry en:fusilli
2020-08-30T22:25:54.5109732Z ERROR - es:camarones already is a synonym of es:camarones for entry en:shrimps - es:gambas cannot be mapped to entry en:prawns
2020-08-30T22:25:54.5110683Z ERROR - es:mermeladas already is a synonym of es:mermeladas-y-confituras for entry en:jams - es:mermeladas cannot be mapped to entry en:marmalades
2020-08-30T22:25:54.5111538Z ERROR - es:mermeladas-de-naranja already is a synonym of es:mermeladas-y-confituras-de-naranja for entry en:orange-jams - es:mermeladas-de-naranja cannot be mapped to entry en:orange-marmalades
2020-08-30T22:25:54.5112372Z ERROR - it:dolcificanti already is a synonym of it:edulcoranti for entry en:sweeteners - it:sostitutivi-dello-zucchero cannot be mapped to entry en:sugar-substitutes
```

**Comment by teolemon** [comment]
I just rebased it.

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -459,11 +467,13 @@ ciqual_food_name:fr:Lait de croissance infantile, liquide (aliment lacté destin
 en:Anti-reflux milk
 ca:Llet contra els refluxs
 fr:Lait anti-reflux
+it:Latte antirigurgito, Latte antiriflusso
```
Comment (VaiTon) [inline taxonomies/categories.txt:470]:
```suggestion
it:Latte antirigurgito, Latte antireflusso
```

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -762,6 +780,7 @@ en:Pollens
 de:Pollen
 es:Polen
 fr:Pollens
+it:Polline
```
Comment (VaiTon) [inline taxonomies/categories.txt:783]:
```suggestion
it:Pollini, polline
```

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -1026,6 +1068,7 @@ de:Craft Cider, Craft Apfelschaumwein, Cidre artisanal
 fi:Artesaanisiiderit
 fr:Cidres artisanaux, Cidre artisanal
 hu:Almaborok
+it:Sidro artigianale, Sidri artigianali
```
Comment (VaiTon) [inline taxonomies/categories.txt:1071]:
```suggestion
it:Sidri artigianali, Sidro artigianale
```

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -1538,6 +1601,7 @@ en:Stout, stout, Dark beers, Dark beer
 de:Stouts
 fi:Stout
 fr:Bières brunes, bière brune
+it:Birra scura
```
Comment (VaiTon) [inline taxonomies/categories.txt:1604]:
```suggestion
it:Birre scure, Birra scura
```

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -1593,6 +1659,7 @@ ciqual_food_name:fr:Bière spéciale (5-6° alcool)
 <en:Beers
 en:5% beer, Beer with 5° alcohol, Beer with 4° alcohol
 fr:Bière à 4-5° d'alcool, Bière à 4%, Bière à 5%
+it:Birra 5%
```
Comment (VaiTon) [inline taxonomies/categories.txt:1662]:
```suggestion
it:Birre 5%, Birra 5%
```

**Review by VaiTon — CHANGES_REQUESTED** [review]
First batch

File: taxonomies/categories.txt [hunk 8]
```diff
@@ -1772,12 +1840,17 @@ origins:en: en:Greece
...
 fr:Tequilas
 hu:Tekilák
 it:Tequila
 nl:Tequilas, Tequila
 nl_be:Tequilas
 ro:Tequilla
+el, el:Ouzo, Oύζο
+protected_name_file_number:en: PGI-CY+GR-01828
+protected_name_type:en: gi
+origins:en: en:Cyprus, Greece
```
Comment (teolemon) [inline taxonomies/categories.txt:1853]:
```suggestion
```

File: taxonomies/categories.txt [hunk 9]
```diff
@@ -1730,10 +1802,6 @@ en:distilled anis
 
 #<en:Cypriot distilled anis
 #<en:Greek distilled anis
```
Comment (teolemon) [inline taxonomies/categories.txt:1804]:
```suggestion
#<en:Greek distilled anis
el, el:Ouzo, Oύζο
protected_name_file_number:en: PGI-CY+GR-01828
protected_name_type:en: gi
origins:en: en:Cyprus, Greece
```

**Comment by teolemon** [comment]
ERROR - he:מוצרי-חלב already is a synonym of he:חלב for entry en:dairies - he:מוצרי-מזון-מותססים cannot be mapped to entry en:fermented-milk-products
ERROR - en:livarot-cheese-from-cow-s-milk already is a synonym of en:livarot for entry en:livarot - en:livarot-cheese-from-cow-s-milk cannot be mapped to entry en:livarot
ERROR - es:leche-condensada already is a synonym of es:leche-condensada for entry en:condensed-milks - es:leches-evaporadas cannot be mapped to entry en:evaporated-milks
ERROR - it:frutta-a-guscio already is a synonym of it:noci for entry en:nuts - it:frutta-a-guscio cannot be mapped to entry en:shelled-nuts
ERROR - he:מדגנים already is a synonym of he:מדגנים for entry en:cereal-grains - he:דגני-בוקר cannot be mapped to entry en:breakfast-cereals
ERROR - it:cereali already is a synonym of it:cereali-e-i-loro-prodotti for entry en:cereals-and-their-products - it:cornflakes cannot be mapped to entry en:corn-flakes
ERROR - de:gepufftes-getreide already is a synonym of de:puffkörner for entry en:puffed-grains - de:gepufftes-getreide cannot be mapped to entry en:puffed-cereals
ERROR - ru:специи already is a synonym of ru:вкусовые-добавки for entry en:condiments - ru:специи cannot be mapped to entry en:spices
ERROR - it:brodo already is a synonym of it:brodo for entry en:broths - it:brodo-liquido cannot be mapped to entry en:liquid-broths
ERROR - de:spiralnudeln already is a synonym of de:spiralnudeln for entry en:torti - de:fusilli cannot be mapped to entry en:fusilli
ERROR - es:camarones already is a synonym of es:camarones for entry en:shrimps - es:gambas cannot be mapped to entry en:prawns

File: taxonomies/categories.txt [hunk 10]
```diff
@@ -40300,14 +41333,17 @@ de:Cornflakes
 es:Copos de maíz tostado
 fi:maissihiutaleet
 fr:Corn-flakes, pétales de maïs, Corn  flakes
+it:Cornflakes, cereali
```
Comment (svensven) [inline taxonomies/categories.txt:41336]:
```suggestion
it:Cornflakes
```

**Review by stephanegigandet — APPROVED** [review]
Looks good to me, thank you

File: taxonomies/categories.txt (hunk 7 above)
Comment (teolemon) [reply taxonomies/categories.txt:1662]: fixed

File: taxonomies/categories.txt (hunk 8 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixed

File: taxonomies/categories.txt (hunk 9 above)
Comment (teolemon) [reply taxonomies/categories.txt:1804]: fixed

File: taxonomies/categories.txt (hunk 6 above)
Comment (teolemon) [reply taxonomies/categories.txt:1604]: fixed

File: taxonomies/categories.txt (hunk 5 above)
Comment (teolemon) [reply taxonomies/categories.txt:1071]: fixed

File: taxonomies/categories.txt (hunk 4 above)
Comment (teolemon) [reply taxonomies/categories.txt:783]: fixed

File: taxonomies/categories.txt (hunk 3 above)
Comment (teolemon) [reply taxonomies/categories.txt:470]: fixed

File: taxonomies/categories.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixed

File: taxonomies/categories.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/categories.txt]: fixed

**Comment by teolemon** [comment]
@VaiTon can you approve, I made the changes

**Pull request merged**